qemu/tests/qemu_disk_img_convert.py

```diff
@@ -80,13 +81,20 @@ def run(test, params, env):
     if not md5:
         test.error("Fail to save tmp file")
     convert_test.destroy_vm()
-    n_params = convert_test.convert()
-    convert_test.compare_test(n_params)
-    convert_test.verify_info(n_params)
-    convert_test.start_vm(n_params)
 
-    # check md5sum after conversion
-    ret = convert_test.check_file(t_file, md5)
-    if not ret:
-        test.error("image content changed after convert")
-    convert_test.clean()
+    cluster_size_set = params.get("cluster_size_set")
+    if cluster_size_set == None:
+        cluster_size_set = ""
+    for cluster_size in re.split(r"\s+", cluster_size_set.strip()):
+        logging.info("convert image file with cluster_size=%s." % cluster_size)
```

Comment:
It is not reasonable if `cluster_size` is `""` for   `  logging.info("convert image file with cluster_size=%s." % cluster_size) `,  it will print the log `convert image file with cluster_size=.`